--- ../../433Utils/CHIP_utils/RFSniffer.cpp	2019-10-14 14:34:48.000000000 +0000
+++ RFSniffer.cpp	2019-10-14 13:17:21.000000000 +0000
@@ -10,20 +10,24 @@
 
 #include "../rc-switch/RCSwitch.h"
 
+
 extern "C" {
 	#include <unistd.h>
+	#include "../rc-switch/CHIP_IO/source/common.h"
+	#include "../rc-switch/CHIP_IO/source/event_gpio.h"
 }
 
 #include <stdlib.h>
 #include <stdio.h>
-     
-     
+
+
 RCSwitch mySwitch;
- 
 
 
 int main(int argc, char *argv[]) {
-  
+
+
+
     /*
      output PIN is hardcoded for testing purposes
      */
